### DISCUSSION TITLE
Keyboard buttons does to be pressed twice when using NVDA

### DISCUSSION THREAD

**Discussion by makhlwf** [nvaccess/nvda#17282] [Q&A]
Hello guys, some of my keyboard buttons starting to being clicked twice. For example, the backspace key. For example, I typed four letters. One click can delete two letters. And the other click can delete the other two letters. When I am navigating in a text document, the arrow keys starting to skip two letters or two lines. I even installed a add-on for NVDA that adds a keyboard sound. And I noticed that I hear the backspace button being clicked two times. So, do you know solution? Just to be sure, the letters is not doing this behavior. All letters from A to Z is clicked one time normally. And other buttons such as Windows, Control, Alt, Applications, Shift, Caps Lock, and some others. But the keyboard buttons is doing this is the ENTER key, which is the NVDA key, and the backspace key, and sometimes the arrow keys. The arrow keys is doing this behavior when I am in a text field. But when I go outside the text field, this behavior stops from the arrow keys. So, please help.

Also when I try to record a voice message and Telegram to clicks cuz the message to start recording and stops recording immediately

The problem is getting worse when I open a web browser

I'm sure that this is not a hardware issue
The Tab Key starts to skip two items as if I clicked it two times

I tried all troubleshooting steps on the Internet, such as disabling sticky keys or filter keys from the settings, but they are disabled by default. And about the keyboard driver, my keyboard is a basic cheap keyboard. It is just a $5 keyboard, so it doesn't have any dedicated driver for it. But I'm sure this is not a hardware issue.

When pressing NVDA plus S in the desktop to change the speech mode, NVDA starts to navigate between speech modes and navigating between the apps that starts with the letter F
S at the same time. This is done at the same time in each click

Doing a complete uninstallation and fresh installation for nvda didn't help.

**Comment by hwf1324** [comment]
Hi,

Can you check the status of the following settings

NVDA Settings -> Keyboard -> Handle keys from other applications

Do you have a programme installed that has a keyboard hook?

Thanks.

**Reply by makhlwf** [reply]
Hello, very thank you for replying. The status of this option is enabled. Also, I'm not sure. This problem is only happen when NVDA is on. I tried using narrator, and this problem doesn't persist. Only when NVDA is enabled. Can you give me examples for well-known applications that have a hook for keyboard? Because I'm not sure if I installed applications that can do this.

**Reply by hwf1324** [reply]
I'm sorry I'm using a software ecology that may be different from yours, so I can't provide much information.

What about restarting after disabling the add-on? Ideally there should also be a debug log of when this behaviour occurs.

**Reply by makhlwf** [reply]
I tried all troubleshooting steps from uninstallation and reinstalling, restarting with all addons disabled, shutdown the system with several ways, checking system settings, but nothing helps. Thank you for watching!
